.travis.yml

```diff
@@ -64,21 +64,14 @@ install:
     pip install --upgrade setuptools
   # Install only from travis wheelhouse
   - if [ -z "$PRE" ]; then
-        wheelhouse_pip_install python-dateutil $NUMPY $PANDAS pyparsing pillow sphinx!=1.3.0;
+        wheelhouse_pip_install python-dateutil $NUMPY $PANDAS pyparsing pillow sphinx!=1.3.0 mock;
```

Comment:
`mock` shouldn't be necessary on Python 3, unless we're using something really new from 3.5.